Assets/BossRoom/Scripts/Shared/ProfileManager.cs

```diff
@@ -0,0 +1,59 @@
+using UnityEngine;
+
+#if UNITY_EDITOR
+using ParrelSync;
+#endif
+
+public static class ProfileManager
+{
+    public const string AuthProfileCommandLineArg = "-AuthProfile";
+
+    static bool s_IsProfileSet = false;
+
+    static string s_Profile = "";
+
+    public static string Profile
+    {
+        get
+        {
+            if (!s_IsProfileSet)
```

Comment:
doing that bool check might be more performant, but do we get Profile that often?
maintainability side, having a != "" might be better? Or even != null